Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css

```diff
@@ -221,10 +221,6 @@ table.queue-grid tr.platform.watchos-8 img.logo {
     content: url(../Images/WatchOS8.png);
 }
 
-table.queue-grid tr.platform.macos-catalina img.logo {
-    content: url(../Images/Catalina.png);
```

Comment:
I'm not sure if this can be removed yet, isn't it used on other instances of the dashboard?

Reply:
I'm pretty sure we redefine this in other instances, and I haven't removed the image. I'll double check before landing.